service/controller/clusterapi/v27/resource/endpoints/desired.go

```diff
@@ -31,7 +31,7 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		// three resync periods max or something.
 		r.logger.LogCtx(ctx, "level", "debug", "message", "master instance not found")
 		resourcecanceledcontext.SetCanceled(ctx)
-		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource reconciliation for custom object")
```

Comment:
Should it remain as `debug` level? I think it's more likely `info`.

Reply:
When resource reconciliation is canceled, are we usually get a notification by logs? (if the default setting is `debug`, I don't have any concern)

Reply:
Maybe look at some operator logs of running operators. There you see how it works. It is the same for all of them.